src/hit/api/params.cpp

```diff
@@ -6,20 +6,20 @@
 using namespace std;
 
 namespace hit {
-    BootstrappingParams::BootstrappingParams(latticpp::BootstrappingParameters btp_params)
-        : lattigo_btp_params(move(btp_params)) {
+    BootstrappingParams::BootstrappingParams(latticpp::BootstrappingParameters lattigo_btp_params)
+        : lattigo_btp_params(move(lattigo_btp_params)) {
     }
 
     int BootstrappingParams::bootstrapping_depth() const {
         return bootstrapDepth(lattigo_btp_params);
     }
 
-    CKKSParams::CKKSParams(latticpp::Parameters params) : lattigo_params(move(params)) {
+    CKKSParams::CKKSParams(latticpp::Parameters lattigo_params) : lattigo_params(move(lattigo_params)) {
     }
 
-    CKKSParams::CKKSParams(latticpp::BootstrappingParameters btp_params)
-        : lattigo_params(genParams(btp_params)),
-          btp_params(optional<BootstrappingParams>(BootstrappingParams(move(btp_params)))) {
+    CKKSParams::CKKSParams(const latticpp::BootstrappingParameters &lattigo_btp_params)
+        : lattigo_params(genParams(lattigo_btp_params)) {
+        btp_params = optional<BootstrappingParams>(BootstrappingParams(lattigo_btp_params));
```

Comment:
Why has this been moved out of the initializer?

Reply:
Refactored this constructor to call another constructor.